Src/ReleaseHistory.md

```diff
@@ -16,6 +16,10 @@
 - UER => eliminate unhandled exceptions in rules
 - UEE => eliminate unhandled exceptions in engine
 
+## Unreleased
+
+- BUG: Resolve unhandled `OutOfMemory` and `NullReferenceException` exceptions when analyzing folders that contain big files even when using --file-size-in-kb. [#621](https://github.com/microsoft/sarif-pattern-matcher/pull/621)
```

Comment:
>Resolve [](http://example.com/codeflow?start=7&length=7)

This note is confusing to me. Is the issue only provoked when using --file-size-in-kb? If not, don't mention it. If it's only provoked using this argument, we don't need the term 'even'. I will review the change and see what I think. :) But this was a little clear on first look. Also, you should always enquote arg names, like `--file-size-in-kb`. I would also suggest using 'large' over 'big', the former term is more typical for API/docs when referring to file sizes.